CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```

Comment:
👍 yup putting this under unreleased is correct. When you do a version bump, you change this to reflect the version number. When you're *sure* it's up on PyPI, you can set the date for the release and create a new Git Tag/Release.